### DISCUSSION TITLE
New guides addition

### DISCUSSION THREAD

**Discussion by jermanuts** [beerisgood/Windows11_Hardening#23] [General]
https://github.com/starchturrets/windows-shenanigans

https://github.com/privacyguides/privacyguides.org/pull/2452

https://github.com/privacyguides/privacyguides.org/pull/1659

**Comment by beerisgood** [comment]
Any specific stuff?

first link isn’t well structured. 
Other two are only issues/ discussions. 

But I’m open for PR.

**Reply by jermanuts** [reply]
Idk tbh, I hate looking at GPs https://devblogs.microsoft.com/oldnewthing/20110606-00/?p=10493

**Reply by beerisgood** [reply]
Well it isn't new that Group Policy's are a mess.
It's even worse that such an article is needed.

**Comment by jermanuts** [comment]
hey @beerisgood should you warn against Microsoft Vulnerable Driver Blocklist and WDAC?

https://infosec.exchange/@wdormann/114105225709232478

**Reply by beerisgood** [reply]
Do I understand correctly that users are protected with activated HVCI? Then everything is okay?
